R/utils.R

```diff
@@ -60,9 +60,7 @@ is_rosetta2 <- function() {
 
 # Returns the type of make command to use to compile depending on the OS
 make_cmd <- function() {
-  # CmdStan 2.21 introduced TBB that requires mingw32-make on Windows
-  ver <- .cmdstanr$VERSION
-  if (os_is_windows() && (is.null(ver) || ver >= "2.21")) {
```

Comment:
This check for versions is not needed. CmdStan 2.20 and earlier will still work with mingw32-make (its just that 2.21 and on requires mingw32-make). The toolchain installation for Windows is made in such a way that users will always have mingw32-make (3.x or 4.x). Not to mention I doubt anyone uses CmdStan 2.20 with CmdStanR :)